tunnel.go

```diff
@@ -97,9 +97,10 @@ type TunnelUpdateParams struct {
 }
 
 type UnvalidatedIngressRule struct {
-	Hostname string `json:"hostname,omitempty"`
-	Path     string `json:"path,omitempty"`
-	Service  string `json:"service,omitempty"`
+	Hostname      string               `json:"hostname,omitempty"`
+	Path          string               `json:"path,omitempty"`
+	Service       string               `json:"service,omitempty"`
+	OriginRequest *OriginRequestConfig `json:"originRequest,omitempty"`
```

Comment:
@jacobbednarz Should I make a new struct like IngressOriginRequestConfig since I believe `ingress.originRequest` is not identical to top-level `originRequest`?
 
https://developers.cloudflare.com/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-guide/local/local-management/ingress/#origin-configuration